### PR TITLE
Add notes about the import process.

### PR DESCRIPTION
**Pull request by mayhem**
Addresses LB-37 and LB-51.

### CODE REVIEW

File: webserver/templates/user/import.html [hunk 1]
```diff
@@ -37,6 +37,22 @@ <h2 class="page-title">{{ user.musicbrainz_id }}</h2>
         </div>
      {% else %}
      <p>
+        <strong>Notes about the Last.fm import process:</strong>
+     </p>
+     <p>
+        The Last.fm importer manually steps through your listen history and imports the listens
```
Comment (alastair) [inline webserver/templates/user/import.html]: Here would be a place to additionally add 

 - Don't go away from this tab when it's running
 - It's better to not be listening/scrobbling at the same time

if wanted

**Comment by alastair** [comment]
:+1:

File: webserver/templates/user/import.html (hunk 1 above)
Comment (mayhem) [inline webserver/templates/user/import.html]: Wanted, but this section is getting big. How about I add them to the second step?

Comment (alastair) [inline webserver/templates/user/import.html]: agreed, it makes more sense there

**Comment by gentlecat** [comment]
:ship:

File: webserver/templates/user/import.html (hunk 1 above)
Comment (mayhem) [inline webserver/templates/user/import.html]: Added LB-50 for this purpose.

**Pull request merged**